src/Lecture1.hs

```diff
@@ -119,4 +126,14 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Int -> [Int] -> String
+lowerAndGreater n list = if null list then "10 is greater than 0 elements and lower than 0 elements" else go 0 0 list
```

Comment:
Do you need to this `if-then-else` at the beginning of the function? I see that in the `go` function you already return the required `String` if the initial list is empty.